### PR TITLE
Added non-dimensional Q criterion derived qty

### PR DESCRIPTION
**Pull request by diederikb**

### CODE REVIEW

**Review by michaeljbrazell — APPROVED** [review]
looks good, let's wait for CI before merging

File: amr-wind/fvm/qcriterion_nondim.H
Comment (sayerhs) [inline amr-wind/fvm/qcriterion_nondim.H:84]: This code looks like a duplicate of the QCriterion class. Can these not be combined together and differentiated at the operator functions that are defined below?

File: amr-wind/fvm/fvm.H
Comment (michaeljbrazell) [inline amr-wind/fvm/fvm.H:12]: I think this line can be deleted now

File: amr-wind/fvm/qcriterion.H
Comment (michaeljbrazell) [inline amr-wind/fvm/qcriterion.H:88]:
```suggestion
                    qcritphi(i, j, k) = 0.5 * (W2/S2-1.0);
```

File: amr-wind/physics/CMakeLists.txt
Comment (michaeljbrazell) [inline amr-wind/physics/CMakeLists.txt:10]:
```suggestion
```

**Pull request merged**
